tron/serialize/filehandler.py

```diff
@@ -57,13 +59,13 @@ def write(self, content):
         """Write content to the fh. Re-open if necessary."""
         if self._fh == NullFileHandle:
             try:
-                self._fh = open(self.name, 'a')
+                self._fh = open(self.name, 'ab')
```

Comment:
I'm kinda afraid that going from text/binary will lead to other unintended consequences.
Does this handle newlines like before?

Reply:
I think newlines are fine, the output from the failing job on stage had newlines.

But I didn't think about cons of binary in general? I guess passing in `encoding=utf-8` instead of changing this to binary would probably fix the issue as well. I can test that.